src/statick_web/plugins/web_tool_plugins/eslint_tool_plugin.py

```diff
@@ -79,17 +108,27 @@ def scan(self, package: Package, level: str) -> Optional[List[Issue]]:
 
     # pylint: enable=too-many-locals
 
+    def remove_config_file(self, install_dir, tool_config):
+        """Remove config file automatically copied into directory."""
+        format_file_path = pathlib.Path(install_dir, tool_config)
+        format_file_path = format_file_path.expanduser()
```

Comment:
I'm fine with how this is, as it is readable. But would the following work and be nicer? (I could go either way).

```
format_file_path = pathlib.Path(install_dir, tool_config).expanduser()
```

Reply:
that does work so i'll switch to that.